client/components/keyed-suggestions/index.jsx

```diff
@@ -172,8 +174,16 @@ class KeyedSuggestions extends Component {
 	onMouseDown = ( event ) => {
 		event.stopPropagation();
 		event.preventDefault();
+
+		const { suggest, recordTracksEvent } = this.props;
 		const suggestion = event.currentTarget.textContent.split( ' ' )[ 0 ];
-		this.props.suggest( suggestion, this.state.isShowTopLevelTerms );
+		if ( this.state.isShowTopLevelTerms ) {
+			recordTracksEvent( 'search_dropdown_searchby_item_click', { term: suggestion } );
```

Comment:
Here the suggestion is `feature:|subject:|column:`, should we trim the semicolon?

Reply:
In my opinion, yes. Let's do it

Reply:
+1, updated!